apps/web/pages/api/auth/forgot-password.ts

```diff
@@ -44,7 +44,7 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
   }
 
   try {
-    const maybeUser = await prisma.user.findUnique({
+    const maybeUser = await prisma.user.findFirst({
```

Comment:
Why can't we use findUnique anymore? Trying to understand.

Reply:
I think it's because email is no longer unique in DB. Same goes for username.

Reply:
It can fetch wrong user because we are getting the first one and not the context specific.
e.g. Acme's John or Deel's John

Reply:
Email is still unique, reverting this change as `findUnique` will still work here: 
<img width="544" alt="image" src="https://github.com/calcom/cal.com/assets/467258/99187e6e-8f62-49be-b2d2-9d5858ab08bf">

Reply:
<img width="1319" alt="Screenshot 2023-06-14 at 11 07 44 AM" src="https://github.com/calcom/cal.com/assets/1780212/b98fec91-44a0-452c-9701-40a60d9d28c4">
So email is no longer unique at DB level. Do we want to make it unique or not?